### DISCUSSION TITLE
Add manual sort capabilities to list-o2m-tree-view interface

### DISCUSSION THREAD

**Discussion by sensedrive** [directus/directus#7316] [Feature Requests]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

Items in the tree interface in a collection are not sortable, althought **branches** collection has sort field and sort field is chosen in the M2A field.

https://user-images.githubusercontent.com/8351587/128881247-5f47b7c4-b7ea-4762-8b64-9968dc7bd202.mov

### To Reproduce

I appended a test dump

login: clwsuepvjembowxpbu@rffff.net
password: 123456

[test-dump-bug.sql.zip](https://github.com/directus/directus/files/6962251/test-dump-bug.sql.zip)

### What version of Directus are you using?

9.0.0-rc.89

### What version of Node.js are you using?

14.17.3

### What database are you using?

mariadb, latest

### What browser are you using?

Firefox

### What operating system are you using?

macOS

### How are you deploying Directus?

running locally with dockered database

**Comment by rijkvanzanten** [comment]
@sensedrive The tree-view o2m interface doesn't currently have the functionality to configure a sort field. I've moved your issue here as a feature request to add that in 👍🏻

**Reply by sensedrive** [reply]
Thank you for pointing that out! I assumed the functionality of sorting would be described through this setting.

<img width="1440" alt="grafik" src="https://user-images.githubusercontent.com/8351587/128885686-b8f90dff-988c-4bc5-92d2-9647d3fd4faf.png">

**Reply by rijkvanzanten** [reply]
Yeah makes sense! Lets see if that's something that's easy to throw in

**Reply by rijkvanzanten** [reply]
It was! Starting with rc.90, it'll work like you expected 🙂

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
